tests/integration/devfile/cmd_devfile_deploy_test.go

```diff
@@ -98,13 +98,13 @@ var _ = Describe("odo devfile deploy command tests", func() {
 	})
 
 	When("using a devfile.yaml containing two deploy commands", func() {
-
 		BeforeEach(func() {
 			helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
 			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile-with-two-deploy-commands.yaml"), path.Join(commonVar.Context, "devfile.yaml"))
 		})
 		AfterEach(func() {
-			helper.Cmd("odo", "v2delete", "-a").ShouldPass()
+			// assuming we are inside the context directory since `odo deploy` cannot be run outside a context directory
+			helper.Cmd("odo", "delete", "component", "-f").ShouldPass()
```

Comment:
IMO, `AfterEach` should help aid cleanup part. If something in it has to have `ShouldPass`, it should either be in the test or we should not have `ShouldPass` call to it. Basically, `AfterEach` should not lead to a test failure, IMO.